app/editor/src/store/hooks/admin/useWorkOrders.ts

```diff
@@ -14,22 +21,24 @@ interface IWorkOrderController {
 }
 
 export const useWorkOrders = (): [IAdminState, IWorkOrderController] => {
-  const api = useApiAdminWorkOrders();
+  const adminApi = useApiAdminWorkOrders();
```

Comment:
Maybe not necessary?

Reply:
I don't think we need `useApiAdminWorkOrders` if we're giving access to editors now.  Move the controller and give appropriate permission.  At some point we'll probably not want editors this kind of access, but for now it'll be okay.